src/moi_oracle.jl

```diff
@@ -232,17 +232,17 @@ function compute_inface_extreme_point(
 	kwargs...,
 ) where {OT, T <: Real}
 	n = size(direction, 1)
-	a = compute_inface_extreme_point(lmo, vec(direction), x)
+	a = compute_inface_extreme_point(lmo, vec(direction), vec(x))
```

Comment:
For better overview, it would be good to move this function before the subroutine function.